model/src/order.rs

```diff
@@ -460,6 +476,42 @@ impl Default for OrderKind {
     }
 }
 
+/// Location for which the sellAmount should be drawn upon order fulfilment
+#[derive(Eq, PartialEq, Clone, Copy, Debug, Deserialize, Serialize, Hash, enum_utils::FromStr)]
+#[enumeration(case_insensitive)]
+#[serde(rename_all = "snake_case")]
+pub enum BalanceFrom {
```

Comment:
Just curious, why not call this `SellTokenBalance` like in the OpenAPI spec? It might remove some confusion that this doesn't specify which account the balance is taken from, but rather how the balance is taken?

I'm also OK with how it is.

Reply:
Well, sure we could do this... but then your next question is going to be why not rename the DB columns names and enums also? Personally I am not a fan of the term "Buy/SellTokenBalance" because it seems like it should be a number, so I would prefer to switch it to something that I feel makes sense as soon as possible internally.

Reply:
I actually would change the column names to:
```
    ADD COLUMN sell_token_balance BalanceFrom NOT NULL default 'erc20',
    ADD COLUMN buy_token_balance BalanceTo NOT NULL default 'erc20';
```

Just because these two are the only columns that don't match the names of their corresponding fields. While its not a big deal, this can be a bit counter-intuitive to someone who joins the project and is trying to figure out "where does the `sell_token_balance` field come from? I can't find a column for it in the database?".

(I glanced over this and missed it in the other review).

Reply:
I belive this is kinda too late now that #898 has already been merged. This would require another DB migration.